cime/src/drivers/mct/main/prep_ocn_mod.F90

```diff
@@ -1066,6 +1110,8 @@ subroutine prep_ocn_merge( flux_epbalfact, a2x_o, i2x_o, r2x_o, w2x_o, g2x_o, xa
                x2o_o%rAttr(index_x2o_Faxa_snow_HDO ,n)
        end if
     end do
+    !Experimental: call subshelf boundary flux code here.  Note this is called on atmospheric timestep.
+    call prep_glc_calculate_subshelf_boundary_fluxes
```

Comment:
This is called from both here and cime_run. Is that duplication intentional? And if this is here intentionally: is there any precedent for calling one component's prep routine from another component's prep routine? I'm concerned that doing this makes it much harder to follow the driver sequencing logic.

Reply:
Removed in 284ab38dc40dd4dae45f093c0d8e391e24864d74.